src/pages/_app.tsx

```diff
@@ -1,17 +1,22 @@
 import { ChakraProvider } from '@chakra-ui/react';
+import { Inter } from '@next/font/google';
 
 import { CoreLayout } from '@/common/components/CoreLayout';
 import { PageHead } from '@/common/components/PageHead';
 import { theme } from '@/common/theme';
 
+const inter = Inter({ weight: 'variable' });
+
 export const App = ({ Component, pageProps }) => {
   const Layout = Component.layout ? Component.layout : CoreLayout;
   return (
     <ChakraProvider theme={theme}>
-      <PageHead />
-      <Layout>
-        <Component {...pageProps} />
-      </Layout>
+      <main className={inter.className}>
```

Comment:
should move this to `line15`

Reply:
sorry for nit picking but can we use `div` instead? otherwise, there will be multiple `main` in the app since it's already used in layout.

Reply:
hold on, im figuring out a way to add multiple fonts to the app